concepts/50 React Components/50 Application Template/45 Authentication/20 Next.js Authentication/20 Implement Authentication Functions.md

```diff
@@ -0,0 +1,18 @@
+Authentication functions for back-end requests are located in `src\app\actions\auth.jsx`. You must update these functions to make requests to your server, such as to add or verify users.
```

Comment:
> You must update these functions to make requests to your server, such as to add or verify users.

Just a small correction: `src/app/actions/` contains server code (take note of the `'use server'` directive), so it's not correct to say "make requests to your server", but rather something about requests to database/authentication infrastructure (maybe the Next.js guide has the right language?)